package.json

```
@@ -22,6 +22,9 @@
     "hook",
     "hooks"
   ],
+  "engines": {
+    "node": ">=12.0.0"
```

```suggestion
    "node": ">=12.20.0"
```

This doesn't fully describe the supported versions. For example, this allow 14.0.0 when one needs 14.3.1 or later on the 14.x line. However, perhaps this is good enough?

The full spelling would be something like `^12.20.0 || ^14.13.1 || ^16.0.0 || ^18.1.0 || >=20.2.0`. That excludes the odd-numbered majors, if that really matters (probably not).